src/openrct2/actions/TrackRemoveAction.cpp

```diff
@@ -43,6 +43,18 @@ uint16_t TrackRemoveAction::GetActionFlags() const
     return GameAction::GetActionFlags();
 }
 
+int32_t TrackRemoveAction::NormaliseTrackType(int32_t trackType) const
```

Comment:
You’re passing `trackType` as a parameter here, but you never use it.

Reply:
oops. fixed.